persistence/src/commonMain/db_user/com/wire/kalium/persistence/MessageDetailsView.sq

```diff
@@ -39,6 +39,10 @@ User.user_availability_status AS senderAvailabilityStatus,
 User.user_type AS senderUserType,
 User.bot_service AS senderBotService,
 User.deleted AS senderIsDeleted,
+User.expires_at AS senderExpiresAt,
+User.defederated AS senderDefederated,
+User.supported_protocols AS senderSupportedProtocols,
+User.active_one_on_one_conversation_id AS senderActiveOneOnOneConversationId,
```

Comment:
just so i understands the issue correctly, we already join the user table so the messages query will be rerun when any user data gets update and also this data get ignored in the view model and it observes the user info again basically running 2 quarries and recomposing the screen 2 times right?